doc/api/modules.markdown

```diff
@@ -425,29 +425,29 @@ in pseudocode of what require.resolve does:
 <!-- type=misc -->
 
 If the `NODE_PATH` environment variable is set to a colon-delimited list
-of absolute paths, then io.js will search those paths for modules if they
+of absolute paths, then Node.js will search those paths for modules if they
 are not found elsewhere.  (Note: On Windows, `NODE_PATH` is delimited by
 semicolons instead of colons.)
 
 `NODE_PATH` was originally created to support loading modules from
 varying paths before the current
-[module resolution](https://iojs.org/api/modules.html#modules_all_together)
+[module resolution](https://nodejs.org/api/modules.html#modules_all_together)
```

Comment:
Can you fix this as well? We should not have absolute paths for intra document references I guess.